### ISSUE TITLE
Do you know if the gyro data are available on miBand 5?

### ISSUE THREAD

**Issue by CarlosBustos1** [NateSchoolfield/blesleep#4]
Hi, I can run your project without problems except the gyro data are not notified by the band. I am using miBand 5 that has firmware 1.0.0.76 . Did you test it with miband5? Thanks

**Comment by TheBeaninator** [comment]
Yes, I've done some testing on the Miband5 and even figured out how to pull
gyro data, but I haven't yet figured out how to correctly parse the bits
coming back.  If you like I can put the Miband5 gyro-starting code into a
separate branch and let you take a crack at decoding them.

**Comment by CarlosBustos1** [comment]
that would be great. I'm going to try to decode the gyroscope data. I wait for the branch

**Comment by lyyiangang** [comment]
@CarlosBustos1  @BeanCounterTop 
https://github.com/BeanCounterTop/miband4/tree/vibrate_and_gyro
